### DISCUSSION TITLE
Feature Request : 'Current world' as an option on the Report Button Plugin

### DISCUSSION THREAD

**Discussion by wizzyhatg** [runelite/runelite#15072] [Ideas]
Hi,

I often do activities involving world hopping where it's helpful to see what world I'm on without opening friend's/world select directly. I currently use a plugin hub plugin (https://github.com/Broooklyn/runelite-external-plugins/tree/current-world) that displays my current world as an infobox, and place the infobox on top of the report button.

However, it looks quite messy with overlapped text and isn't the best solution. Would it be possible to add an option for the report button to display the current world similar to how it can currently display UTC time, time logged in etc.?

As a simpler alternative, maybe an option for the report button to just not have any text on it so it can be a good spot to place any external plugin infobox?

My current scuffed solution:
![explorer_2022-06-05_14-34-19](https://user-images.githubusercontent.com/39323010/172069485-b7a27de4-5436-43e0-8fce-bb27ee9455ac.png)
 
Proposed mockup:
![explorer_2022-06-05_14-35-00](https://user-images.githubusercontent.com/39323010/172069523-2f545e3c-7a3b-4cce-ba22-fd3828e8636f.png)

**Comment by wizzyhatg** [comment]
I am not sure how to/ if I should have added a feature request label, and am not sure how to edit. I do not use Github much.

**Reply by pwatts6060** [reply]
I think it would be better to add it in the title bar as a toggle so it would say:
**Runelite - Username - W301**

Report button is small and has too many other useful options
